src/main/scala/atom/Atom.scala

```diff
@@ -0,0 +1,113 @@
+package laughedelic.atom
+
+import scala.scalajs.js, js.annotation._
+
+@js.native
+trait Disposable extends js.Object {
+  def dispose(): Unit = js.native
+}
+
+@js.native
+trait CommandRegistry extends js.Object {
+  def add(target: String, commandName: String, callback: js.UndefOr[js.Any => js.Any]): Unit = js.native
+  def add(target: String, commands: js.Dictionary[js.Any => js.Any], callback: js.UndefOr[js.Any => js.Any]): Unit = js.native
```

Comment:
I'm not sure about the second `add` method. I'm looking at the [Atom API docs](https://atom.io/docs/api/v1.23.3/CommandRegistry#instance-add) ("Registering Multiple Commands" part) and it doesn't have the `callback` parameter. Or am I looking in a wrong place? 🤔 

Plus both `add` versions should return a `Disposable`.

Reply:
You are right! I't my fault I took some of the definitions from https://github.com/atom/atom-languageclient/blob/master/flow-libs/atom.js.flow.
I forgot to `Disposable` but I was aware of that.